ivi-layermanagement-api/ilmControl/src/ilm_control_wayland_platform.c

```diff
@@ -2121,26 +2124,43 @@ static void screenshot_done(void *data, struct ivi_screenshot *ivi_screenshot,
         return;
     }
 
+    // Store the image in image_buffer in the follwing order B, G, R, [A](B at the lowest address)
     for (row = 0; row < height; ++row) {
```

Comment:
those for loop seam only be required for bmp dumps, for the png we should be able to control with with the newly added _set_bgr call, right?
and do we skip the alpha for png? this should not be done! please check if the png image will also contain the alpha values!

Reply:
I moved this loop inside the specific format writing functions, so that each function reorder the raw image as needed.